Sources/IO/Misc/GCodeReader/example/index.js

```diff
@@ -0,0 +1,137 @@
+import '@kitware/vtk.js/favicon';
+
+// Load the rendering pieces we want to use (for both WebGL and WebGPU)
+import '@kitware/vtk.js/Rendering/Profiles/Geometry';
+
+import vtkFullScreenRenderWindow from '@kitware/vtk.js/Rendering/Misc/FullScreenRenderWindow';
+import vtkActor from '@kitware/vtk.js/Rendering/Core/Actor';
+import vtkMapper from '@kitware/vtk.js/Rendering/Core/Mapper';
+import vtkGCodeReader from '@kitware/vtk.js/IO/Misc/GCodeReader';
+
+import controlPanel from './controller.html';
+
+// ----------------------------------------------------------------------------
+// Example code
+// ----------------------------------------------------------------------------
+
+let renderer = null;
+let renderWindow = null;
+let layersSelector = null;
+let layersLabel = null;
+
+const reader = vtkGCodeReader.newInstance();
+
+// ----------------------------------------------------------------------------
+
+function hslToRgb(h, s, l) {
```

Comment:
Don't we have something like that in vtkMath?

Reply:
Yes, fixed the example